### DISCUSSION TITLE
Where's sounds of weapons in cyberpunk?

### DISCUSSION THREAD

**Discussion by BouncyBearVII** [WolvenKit/WolvenKit#1045] [General]
I cant find the sounds.
What's the folder in the archive? I need to find sounds of reload, inspections and sounds of shot

**Comment by Zhincore** [comment]
Hii, nearly two years later I made [SoundDB](https://sounddb.redmodding.org/) along with [a guide on Wiki](https://wiki.redmodding.org/cyberpunk-2077-modding/modding-guides/sound/finding-an-audio-file).
Necro-posting in case someone is still searching :D
